bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClient.java

```diff
@@ -648,10 +648,11 @@ public void loadDataIntoTable(
       List<String> sourceUris,
       FormatOptions formatOptions,
       JobInfo.WriteDisposition writeDisposition,
-      Optional<Schema> schema) {
+      Optional<Schema> schema,
+      TableId tableToWrite) {
```

Comment:
nit: rename to `destinationTable` or similar

Reply:
Done